tests/integration/model_providers/huggingface/test_huggingface.py

```diff
@@ -315,3 +316,51 @@ def test_hf_model_runner(self, execution_mechanism):
             )
         finally:
             server.wait_for_completion()
+
+    @pytest.mark.parametrize(
+        "execution_mechanism",
+        ["naive", "process_pool", "dedicated_process", "thread_pool"],
+    )
+    def test_hf_custom_model_runner(self, execution_mechanism):
+        project = mlrun.new_project("test-hf-model", save=False)
+        self.setup_datastore_profile(task="image-classification")
+        model_url = self.url_prefix + self.image_classification_model
+        model_artifact, llm_prompt_artifact, function = setup_remote_model_test(
```

Comment:
please add a comment regarding the source of the image - and note that it is free to use.